src/pybind/matrix_reader.py

```diff
@@ -0,0 +1,89 @@
+import kaldi_pybind as kp
+
+class SequentialVectorReader(kp.SequentialBaseFloatVectorReader):
+    def __init__(self, rspecifier = None):
```

Comment:
no spaces before or after = if it appears in the function formal/actual argument
```python
def __init__(self, rspecifier=None)
```